packages/bpk-docs/src/layouts/NeoSideNav/NavList.js

```diff
@@ -32,9 +32,18 @@ import {
 const getClassName = cssModules(STYLES);
 
 const NavLink = (props: LinkPropType) => {
+  const handleClick = () => {
```

Comment:
The handleClick stuff here isn't working. I think we need to somehow close the menu when a selection is made, but not sure how best to go about this. At the moment I've got an issue passing the onSelect prop
![screen shot 2018-04-07 at 11 03 12](https://user-images.githubusercontent.com/30267516/38453799-56362ee6-3a53-11e8-9b6a-7546753605a6.png)